opm/autodiff/BlackoilPropsAdFromDeck.cpp

```diff
@@ -142,6 +143,7 @@ BlackoilPropsAdFromDeck::BlackoilPropsAdFromDeck(const BlackoilPropsAdFromDeck&
     // For data that is dependant on the subgrid we simply allocate space
     // and initialize with obviously bogus numbers.
     cellPvtRegionIdx_.resize(number_of_cells, std::numeric_limits<int>::min());
```

Comment:
Yes it will be communicated. Putting bogus numbers there eases debugging if something goes wrong. 

I guess the entries will be zero initialized anyway, at least if they are real objects.